skemail-web/app.html

```diff
@@ -33,11 +20,12 @@
         }
 
         if (scripts.length) {
-          document.write(
-            scripts
-              .map((script) => `<script defer src="${script}"><\/script>`)
```

Comment:
isn't `defer` also set here?

Reply:
Yes, it is. Both scripts have been deferred. Let me change that in the instructions so that it is made clear that that is not changed.